### ISSUE TITLE
Season of Discovery error after 1.5.4 update

### ISSUE THREAD

**Issue by omggga** [evil-morfar/RCLootCouncil_Classic#60]
There is error now while trying to open /rc config settings:

```lua
8x RCLootCouncil_Classic/RCLootCouncil/core.lua:378: attempt to call global 'InterfaceOptionsFrame_OpenToCategory' (a nil value)
[string "@RCLootCouncil_Classic/RCLootCouncil/core.lua"]:378: in function `?'
[string "@Atlas/Libs/AceConsole-3.0-7/AceConsole-3.0.lua"]:90: in function `?'
[string "@Blizzard_ChatFrameBase/Classic/ChatFrame.lua"]:5034: in function <Blizzard_ChatFrameBase/Classic/ChatFrame.lua:4980>
[string "=[C]"]: in function `ChatEdit_ParseText'
[string "@Blizzard_ChatFrameBase/Classic/ChatFrame.lua"]:4686: in function <Blizzard_ChatFrameBase/Classic/ChatFrame.lua:4685>
[string "=[C]"]: in function `ChatEdit_SendText'
[string "@Blizzard_ChatFrameBase/Classic/ChatFrame.lua"]:4722: in function `ChatEdit_OnEnterPressed'
[string "*ChatFrame.xml:120_OnEnterPressed"]:1: in function <[string "*ChatFrame.xml:120_OnEnterPressed"]:1>
```

**Comment by evil-morfar** [comment]
Yeah, already fixed in 123b913eb5d2916e195731bb28d4c259e976177a. Thanks for letting me know though.
Still have to add the new items from the new phase, won't have time for that before tomorrow.

You can still access it through the normal options -> AddOns menu.

**Comment by omggga** [comment]
Great, thanks!
